### ISSUE TITLE
[BUG] Wrong keybind shown for Pyre as long as there is a keybind for Actionbar 1, button 1

### ISSUE THREAD

**Issue by ivancuric** [Hekili/hekili#2771]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

I have found a similar issue before, and I'm not sure that this fits the description, or the category (recommendations), but I feel that snapshots might help.

Ever since I created a different profile for Aug evoker, the Pyre spell is stuck on the "Q" keybinding for some reason, or whatever the keybinding is for Actionbar 1 button 1.

I've tried everything, and the only thing that helps is removing the keybinding for that button, or manually adding the keybind text for Pyre in Hekili's options.

![image](https://github.com/Hekili/hekili/assets/2827047/462d0f96-8e5e-4c47-8f1b-e5104459db51)
![image](https://github.com/Hekili/hekili/assets/2827047/e4345560-665e-4b2b-a668-ea53019f086d)

### How to Reproduce

1.  Set up a spec/hekili mode so that Pyre gets recommended
2. Place Pyre on any bar and set any keybind, or even remove it.
3. Place an ability on Actionbar 1, button 1, and set a keybind
4. The same keybind will be recommended for both the correct ability on bar1-1, and Pyre which doesn't even have a keybind. 

### Snapshot (Link)

https://pastebin.com/s7AGfmFr

**Comment by Hekili** [comment]
I suspect your action bar numbers or paging are different than you intend.

`pyre                   = Q   [02], 3   [06]`

You have Pyre on 2 action bars (or pages).  It's bound to Q on bar 2 and bound to 3 on bar 6.  Bars 6 and higher (IIRC) also use bindings from action bar 1 to match the default behavior of the game.

The simplest solution is to just manually set the Pyre keybind in /hekili > Abilities > Override Keybind Text, if you don't want to mess with your bars and bindings.

**Comment by ivancuric** [comment]
This is weird as I most certainly don't have it on bar 6, and I'm not using any other 3rd party action bar addons.

I already did the override but would love to get to the bottom of this.
![image](https://github.com/Hekili/hekili/assets/2827047/449493bb-445f-43ca-9af8-77a6184c8cd6)
![image](https://github.com/Hekili/hekili/assets/2827047/84380d8d-237e-4aaf-af29-394cbfc7ef6a)

[EDIT]
Ok, toggled action bar paging, there was one pyre on page 2. After I deleted it - I couldn't get back to my old bar until I reloaded the UI. Weird stuff.
